core/domain/skill_domain_test.py

```diff
@@ -34,9 +34,11 @@ def setUp(self):
         super(SkillDomainUnitTests, self).setUp()
         skill_contents = skill_domain.SkillContents(
             state_domain.SubtitledHtml(
-                '1', '<p>Explanation</p>'), [
-                    state_domain.SubtitledHtml('2', '<p>Example 1</p>')],
-            {'1': {}, '2': {}}, state_domain.WrittenTranslations.from_dict(
+                '1', 'Explanation'), [
```

Comment:
Same question about p tags.

Reply:
Ah, I delete this while resolving the merge conflict. I'll fix it.